crowbar.yml

```diff
@@ -73,6 +73,9 @@ nav:
     logs:
       order: 10
       route: 'utils_path'
+    backup:
+      order: 20
+      route: 'backup_path'
```

Comment:
makes sense somehow. but I would rather not drop the link completely. Maybe we can just add a flash message which says sth like "THis Backup Page is meant only to assist the Upgrade to Cloud 6" ?